### ISSUE TITLE
request: store database schema version in the database, so we can see if we need to upgrade the schema

### ISSUE THREAD

**Issue by bjmgeek** [spl0k/supysonic#95]
It would be relatively simple to add another table, called schemaversion.  That could have one row in it, with a single value (integer or text or whatever) that would be upgraded if the schema version changes.  Then, the user could know (or we could detect programmatically) and apply the upgrade.

**Comment by spl0k** [comment]
Hi.

I've added some automatic database installation and migration system. So now you don't have to run SQL scripts to create the initial tables or upgrade the schema. I ran some tests on it, but not so extensively. Hopefully it won't break anything.
It should work with already existing databases, but if you have any unapplied migration they should be applied manually to catch up with the latest schema version. Future migrations will be handled automatically.

Docker image maintainers (poke @ned-kelly @ogarcia @foosinn @mikafouenski @oakman @glogiotatidis) might want to update their startup scripts.
